### PR TITLE
fix for install/startup bugs caused by package getting installed before ...

### PR DESCRIPTION
**Pull request by alexmbird**
Installing the package before config files meant that a restart was required when the templates were written and they changed.

But if you changed the nodename this restart was broken.  Rabbitmqctl would try to find a server on the new nodename to stop and not find one, then try to start the new one which would fail because the daemon was already running.

Writing configs before installing the package fixes this.

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -21,6 +22,55 @@
...
+    group 'rabbitmq'
+    mode '775'
+    recursive true
+  end
+end
+
+directory node['rabbitmq']['mnesiadir'] do
+  owner 'rabbitmq'
+  group 'rabbitmq'
+  mode '775'
```
Comment (cmluciano) [inline recipes/default.rb]: Can you change this to 00755?

**Comment by cmluciano** [comment]
@alexmbird Are you still interested in this patch? If so can you address the comments?

File: recipes/default.rb (hunk 1 above)
Comment (michaelklishin) [inline recipes/default.rb]: We've changed it to `750` in RabbitMQ packages recently because some users can't have a world-readable data directory.

Comment (cmluciano) [inline recipes/default.rb]: Cool so let's go with that @alexmbird

**Comment by cmluciano** [comment]
Closing due to inactivity

**Pull request closed without merging**